src/public/pages/scripts/features/templates/folder_browser.html

```
@@ -0,0 +1,13 @@
+<div class="modal-box w-11/12 max-w-2xl">
+	<h2 class="text-lg font-bold" data-i18n="util.folderBrowser.title"></h2>
+	<div class="py-2 flex items-center gap-2">
+		<input class="input input-sm flex-1" id="folder-path-input" data-i18n="util.folderBrowser.path" user-content />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)